acceptance/tests/i18n/modules/puppet_agent.rb

```diff
@@ -1,7 +1,7 @@
 test_name 'C100565: puppet agent with module should translate messages' do
   confine :except, :platform => /^solaris/ # translation not supported
 
-  tag 'audit:high',
```

Comment:
This isn't strictly necessary in `main` due to the `skip_test` below, but setting it to `medium` makes it easier to backport to 7.x